### ISSUE TITLE
disable-services interferes with Sysprep

### ISSUE THREAD

**Issue by Tuxhedoh** [W4RH4WK/Debloat-Windows-10#87]
It appears that the dmwappushservice needs to be started in order for sysprep to complete properly. 
I followed these directions to resolve it:
1.Type the following command to set the startup type:
sc config dmwappushservice start= delayed-auto

2.Type the following command to start the service:
sc start dmwappushservice

**Comment by W4RH4WK** [comment]
Can you please state more specifically what this issue is about.

**Comment by Tuxhedoh** [comment]
In building a Windows 10 - Enterprise machine, I deployed the image via MDT, entered a pause sequence, during which time I ran most of the debloat scripts.  Upon re-starting the task sequence, it moves into the sysprep step... and it hangs during the generalizing image.  It will hang indefinitely ( at least a few hours ).  Upon investigating I discovered that there was some need for this service to be running in order for the sysprep/generalize to complete.  As soon as the dmwappushservice is started, generalize completes and the machine is sealed... and the MDT task sequence continues.

This seems to refer to my particular problem... though I'm not using referenced build, but 1607
https://social.technet.microsoft.com/Forums/en-US/618f4531-f824-476d-9327-f9f3f4f8d0b4/slow-sysprep-and-first-logon?forum=win10itprosetup
